redisinsight/api/provider-interfaces/models/agreements.ts

```diff
@@ -0,0 +1,14 @@
+export class Agreements {
+  // as a userId
+  id: string; // todo: Discuss
```

Comment:
What is the meaning of an `id` of an agreement (that is distinct from a `userId`) ?